src/components/settings/SafeModules/index.tsx

```diff
@@ -18,14 +14,6 @@ const NoModules = () => {
   )
 }
 
-const getModuleName = (chainId: string, address: string): string => {
-  if (sameAddress(getSpendingLimitModuleAddress(chainId), address)) {
-    return 'Spending limit module'
-  }
```

Comment:
Wasn't this quite useful?

Reply:
Maybe we could initialize the AB with some preset address like this?

Reply:
I like the idea but I think it's best backend decide the known addresses. It means there isn't a populated address book from the get go as well. What do you think?

Reply:
Backend having it would be best of course.

Reply:
we only need somebody with permissions on prod to add the module to the known addresses list